package.json

```diff
@@ -1,12 +1,12 @@
 {
+  "scripts": {
+    "install": "bower install",
```

Comment:
Does this work? I thought it had to be `postinstall`.

Reply:
![image](https://user-images.githubusercontent.com/4219105/58121632-dac36600-7bd5-11e9-9d6b-2d28231a8c67.png)

🤷‍♂

Reply:
TIL: `install, postinstall: Run AFTER the package is installed.`
https://docs.npmjs.com/misc/scripts